courses/views.py

```diff
@@ -45,3 +46,16 @@ def list(self, request, uuid_uuid, *args, **kwargs):
         serializer = self.serializer_class(
             modules, many=True, context={'request': request})
         return Response(serializer.data)
+
+
+@api_view()
+def user_existence(request):
+    """
+    Support for validating the existence of users, ensuring that they
+    represent a real edx author.
+    """
+    uid = request.GET.get('uid')
+    if not uid:
+        return Response({"error": "Must provide a UID"}, status=400)
+
+    return Response({"exists": Course.objects.filter(instructors__edx_uid=uid).exists()})
```

Comment:
It seems like this endpoint should just check for the existence of an `EdxAuthor`, otherwise it confuses user existence with course membership